python/ucxx/ucxx/_lib/tests/test_probe.py

```diff
@@ -128,7 +128,6 @@ def test_message_probe(transfer_api):
     server.start()
     client = mp.Process(target=_client_probe, args=(queue, transfer_api))
     client.start()
-    client.join(timeout=10)
-    server.join(timeout=10)
+    join_processes([client + server], timeout=10)
```

Comment:
**Problem**: The `join_processes` function is called with an incorrect argument. The function expects a list of processes, but the code is trying to add two processes together (`client + server`).

**Suggested Change**: Pass the processes as a list to the `join_processes` function.


**Severity (1 - 4)**: 2 - MINOR

**Line**: 131

```suggestion
    join_processes([client, server], timeout=10)
```

-----

<div align="right">
    <small><i>Generated by <b>Llama 3.1 405b</b></i></small>
</div>
<br />


<div align="right">
  <small>Was this helpful?</small>
  <a href="https://codecritic-feedback.nvidia.com?vote=1&comment_guid=3c7590dd-273a-4787-bfe2-aa729e6d66d0&is_github=true" target="_blank">
    <button>👍</button>
  </a>
  <a href="https://codecritic-feedback.nvidia.com?vote=-1&comment_guid=3c7590dd-273a-4787-bfe2-aa729e6d66d0&is_github=true" target="_blank">
    <button>👎</button>
  </a>
</div>